crates/hyperswitch_connectors/src/connectors/jpmorgan.rs

```diff
@@ -408,9 +690,10 @@ impl ConnectorIntegration<Execute, RefundsData, RefundsResponseData> for Jpmorga
     fn get_url(
         &self,
         _req: &RefundsRouterData<Execute>,
-        _connectors: &Connectors,
+        connectors: &Connectors,
     ) -> CustomResult<String, errors::ConnectorError> {
-        Err(errors::ConnectorError::NotImplemented("get_url method".to_string()).into())
+        let endpoint = self.base_url(connectors);
+        Ok(format!("{}/refunds", endpoint))
```

Comment:
Ok(format!("{}/refunds", self.base_url(connectors)))

Reply:
okay, removing the var `endpoint`